### DISCUSSION TITLE
Strongswan events or log for IPSec packet drops at kernel

### DISCUSSION THREAD

**Discussion by hnsantu554** [strongswan/strongswan#2454] [Q&A]
How to enable logs for packet drop at kernel level. If a packet dropped due to missing IPSec header how do we get to know? Any events available in strongswan? Please help me to find out thank you

**Comment by tobiasbrunner** [comment]
Check the counters in `/proc/net/xfrm_stat`. There might also be an audit message from the kernel (there is definitely no regular log message).

**Reply by hnsantu554** [reply]
Thank you
